object-construction-checker/tests/mustcall/MustCallChoiceExamples.java

```diff
@@ -0,0 +1,54 @@
+// Simple tests of @MustCallChoice functionality on wrapper streams.
+
+import java.io.*;
+import java.net.*;
+import org.checkerframework.checker.objectconstruction.qual.*;
+import java.io.IOException;
+import org.checkerframework.checker.calledmethods.qual.*;
+
+class MustCallChoiceExamples {
+
+    void test_two_locals(String address) {
+        Socket socket = null;
+        try {
+            socket = new Socket( address, 80 );
+            DataInputStream d = new DataInputStream(socket.getInputStream());
+        } catch (IOException e){
+
+        } finally {
+            closeSocket(socket);
+        }
+    }
+
+    // :: error: required.method.not.called
```

Comment:
We should not get an error here!  Why can't our tool prove this one safe?  (Note that `new DataInputStream(b)` cannot throw an `IOException`)

Reply:
you're right. I fixed it.